core/src/main/java/eu/tsystems/mms/tic/testframework/execution/testng/worker/finish/MethodContextUpdateWorker.java

```diff
@@ -53,8 +53,14 @@ public void onMethodEnd(MethodEndEvent event) {
         // Handle assertions and exceptions in dataprovider methods
         if (testResult.getMethod().isDataDriven()
                 && methodContext.readErrors().anyMatch(ErrorContext::isNotOptional)) {
-            testResult.setStatus(ITestResult.SKIP);
-            methodContext.setStatus(Status.SKIPPED);
+            /**
```

Comment:
JavaDoc has to be outside method. Suggestion: Move whole comment to declaration.

Reply:
Simplified the comment